nixos/modules/services/misc/paperless.nix

```diff
@@ -221,15 +224,18 @@ in
       };
       environment = env;
       wantedBy = [ "multi-user.target" ];
-      wants = [ "paperless-ng-consumer.service" "paperless-ng-web.service" ];
+      wants = [ "paperless-consumer.service" "paperless-web.service" ];
 
       preStart = ''
-        ln -sf ${manage} ${cfg.dataDir}/paperless-ng-manage
+        ln -sf ${manage} ${cfg.dataDir}/paperless-manage
+        ${optionalString hasOldSystemState ''
+          ln -sf ${manage} ${cfg.dataDir}/paperless-ng-manage
+        ''}
```

Comment:
Scripts using the old path will fail.
But because `hasOldSystemState` is no longer needed for redis, we should make a minor backwards-incompatible change and remove the alias. I've noted this in the release notes.